rfc/cart-api-v1.md

```diff
@@ -0,0 +1,84 @@
+# Version 1
+
+| Issue | Status |
+| ----- | ------ |
+| #88   | Draft  |
+
+## Data Models
+
+### `Cart`
+| Property | Type | Description |
+| --- | --- | --- |
+| `id` | UUID-like `str` | Cart ID, auto-generated by the web API |
+| `user_id` | UUID-like `str` | The ID of the user that creates the collection |
+| `cart_name` | str | Cart Name
+| `entity_type` | str | Entity Type |
+
+### `CartItem`
+| Property | Type | Description |
+| --- | --- | --- |
+| `id` | UUID-like `str` | Cart Item ID, auto-generated by the web API |
```

Comment:
Cart items should not have their own id. The tuple `(cart_id, entity_id, bundle_uuid, bundle_version)` uniquely identifies a cart item.

Reply:
So, in this case, `id` is a string representation of `sha1(cart_id, entity_id, bundle_uuid, bundle_version)`?